tools/changelog.py

```diff
@@ -76,9 +76,18 @@ def normalize_title(title):
     See the test suite for the potential matches.
     GitHub sends us a unicode string.
     """
-    if '🚀' in title:
-        title = title.replace('🚀', '')
-        title = title.strip()
+    # if '🚀' in title:
+    #     title = title.replace('🚀', '')
+    #     title = title.strip()
+    #     title = 'NPM update - {title}.'.format(title=title)
+
+    if 'Bump' in title:
```

Comment:
ok this is the core of the code. While 🚀 was probably a unique character I have the feeling it will not be the case for the `Bump` word, aka there could be something else called Bump. So probably the code needs to be a bit more specific. Maybe the code should target the beginning of the string with [startswith](https://docs.python.org/3/library/stdtypes.html#str.startswith)

Reply:
> ok this is the core of the code. While rocket was probably a unique character I have the feeling it will not be the case for the `Bump` word, aka there could be something else called Bump. So probably the code needs to be a bit more specific. Maybe the code should target the beginning of the string with [startswith](https://docs.python.org/3/library/stdtypes.html#str.startswith)

True, I will change that.